apps/web/src/components/ChatWrapper/ErrorMessage/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { captureException } from '$/helpers/captureException'
+import { Alert } from '@latitude-data/web-ui/atoms/Alert'
+import { useEffect } from 'react'
+
+export function ErrorMessage({ error }: { error: Error }) {
+  useEffect(() => {
+    captureException(error)
+  }, [error])
```

Comment:
Do we want to send Sentry chat error messages?

Reply:
for now yes, we'll see what we get